### PR TITLE
Add Network

### PR DESCRIPTION
**Pull request by AxelVivienCosmo**

### CODE REVIEW

File: KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Network.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ */
+
+package io.cosmosoftware.kite.instrumentation;
+
+import io.cosmosoftware.kite.exception.KiteTestException;
+import io.cosmosoftware.kite.report.Status;
+
+import javax.json.JsonObject;
+
+public class Network {
```
Comment (ManuCosmo) [inline KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Network.java]: rename to Profile

I think @sajidcosmo need to make this an OGM entity to match with the model Profile

and merge NWCommand into this class

File: KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/NWCommands.java [hunk 2]
```diff
@@ -12,71 +12,62 @@
 public class NWCommands {
```
Comment (ManuCosmo) [inline KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/NWCommands.java]: Merge this into Profile class

File: KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Networks.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+import io.cosmosoftware.kite.exception.KiteTestException;
+
+import javax.json.JsonArray;
+import java.util.HashMap;
+
+/**
+ * The type Networks.
+ */
+public class Networks extends HashMap<String, Network> {
```
Comment (ManuCosmo) [inline KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Networks.java]: rename to NetworkProfile

I think @sajidcosmo need to make this an OGM entity to match with the model NetworkProfile

**Comment by AxelVivienCosmo** [comment]
Not ready to merge yet since cleanUp has issue if user use command instead of delay, packetloss etc ... 

Commands sent to KiteServer have not been tested yet

Need to make profile an abstract class as you asked previously

**Comment by AxelVivienCosmo** [comment]
But most of the logic is there and I could use feedback on it

File: KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/NetworkProfiles.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+  private final String name;
+
+
+  /**
+   * Instantiates a new Instance.
+   *
+   * @param jsonObject the json object
+   */
+  public NetworkProfiles(JsonObject jsonObject) throws KiteTestException {
```
Comment (ManuCosmo) [inline KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/NetworkProfiles.java]: no s

File: KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Networks.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+import io.cosmosoftware.kite.exception.KiteTestException;
+
+import javax.json.JsonArray;
+import java.util.HashMap;
+
+/**
+ * The type Networks.
+ */
+public class Networks extends HashMap<String, NetworkProfiles> {
```
Comment (ManuCosmo) [inline KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Networks.java]: NetworkProfileHashMap

File: KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Profiles.java [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+ */
+
+package io.cosmosoftware.kite.instrumentation;
+
+import io.cosmosoftware.kite.exception.KiteTestException;
+import io.cosmosoftware.kite.report.Status;
+
+import javax.json.JsonObject;
+
+public class Profiles {
```
Comment (ManuCosmo) [inline KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Profiles.java]: no s

File: KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Scenario.java [hunk 7]
```diff
@@ -8,147 +8,194 @@
...
+          result.append(KiteServerCommand(url));
+        } else if (instrumentUrl.startsWith("http://")) {
+          String url = instrumentUrl + GridId + "&ip=" + nodeIp + "&command=" + this.command;
+          result.append("via KiteServer ").append(KiteServerCommand(url));
+        } else {
+          result.append(" failed, check instrumentUrl !");
         }
-        result.append("\n\n");
+      } else {
+        result.append("Sorry node ").append(nodeIp).append(" is not Linux");
```
Comment (ManuCosmo) [inline KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Scenario.java]: dont be sorry

File: KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Scenario.java [hunk 8]
```diff
@@ -8,147 +8,194 @@
...
   }
 
-  public Integer getDuration() {
-    return duration;
+  public String getType() {
+    return type;
   }
 
-  public String runCommands() {
+  public String sendCommand(WebDriver webDriver, Instrumentation instrumentation, String remoteAddress, String GridId, String instrumentUrl) {
```
Comment (ManuCosmo) [inline KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Scenario.java]: too much repeated code. pls create reusable functions and use them in sendCommand and cleanUp

File: KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Scenario.java [hunk 9]
```diff
@@ -8,147 +8,194 @@
...
+    } catch (Exception e) {
+      result = "thrown an ERROR";
+    }
+    return result;
   }
 
+  private String KiteServerCommand(String url) {
+    String result;
+    try {
+      Desktop desktop = java.awt.Desktop.getDesktop();
```
Comment (ManuCosmo) [inline KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Scenario.java]: pls dont tell me your using a browser to send the HTTP command...

**Comment by ManuCosmo** [comment]
pls fix the conflicts

**Pull request closed without merging**